RELEASE.md

```diff
@@ -1 +1,64 @@
-The release notes have been moved to the ['Releases' section](https://github.com/GPflow/GPflow/releases) of the GPflow GitHub Repo
+Release notes for past releases are available in the ['Releases' section](https://github.com/GPflow/GPflow/releases) of the GPflow GitHub Repo.
+
+# Release x.y.z
```

Comment:
can we not have the template at the bottom of the page?

Reply:
We could, but that makes it more hassle when creating a new release to copy things from the bottom back to the top - to me it makes more sense to follow TensorFlow in this and leave as is.